.swiftlint.yml

```diff
@@ -83,20 +83,21 @@ excluded:
   - "**/Package.swift"
   - "**/Package@swift-*.swift"
   - Carthage
-  - "WireUI/Sources/WireDesign/Icons/Autogenerated/StyleKitIcons.generated.swift"
-  - "WireUI/Sources/WireDesign/Icons/Autogenerated/WireStyleKit.swift"
+  - DerivedData
```

Comment:
For some reason specifying `**/DerivedData` doesn't prevent SwiftLint to exclude any `DerivedData` folder at the root.